test/unit/lint/test-dependencies.js

```diff
@@ -32,8 +32,13 @@ describe('dependencies', () => {
       `app/ui${all}${valid}`,
     ];
 
+    const implicit = [
+      // These aren't specified in package.json but the are installed anyway.
+      'fbjs',
```

Comment:
This is a little odd - I'm not sure about if we should be doing this or instead explicitly relying on fbjs in package.json.  Feel free to land for now to clear out your pending patch list but @linclark what do you think about this?